package.json

```diff
@@ -111,6 +111,7 @@
   "scripts": {
     "lint": "jshint . && node bin/jscs lib test bin publish",
     "test": "npm run lint && mocha",
+    "test-appveyor": "mocha --color",
```

Comment:
There is a lot of comments here now, so i lost track of why this is needed, i'd like to fully understand why it is required to have `--color` pseudo-option in appveyor but not in travis.